src/sentry/integrations/jira/notify_action.py

```diff
@@ -5,34 +5,96 @@
 from django import forms
 
 from sentry.rules.actions.base import IntegrationEventAction
+from sentry.models import ExternalIssue
 
 logger = logging.getLogger("sentry.rules")
 
 
 class JiraNotifyServiceForm(forms.Form):
-    # TODO 1.0 Add form fields.
+    jira_project = forms.ChoiceField(choices=(), widget=forms.Select())
+    issue_type = forms.ChoiceField(choices=(), widget=forms.Select())
 
     def __init__(self, *args, **kwargs):
+        projects_list = [(i.id, i.name) for i in kwargs.pop("integrations")]
         super(JiraNotifyServiceForm, self).__init__(*args, **kwargs)
 
+        if projects_list:
+            self.fields["jira_project"].initial = projects_list[0][0]
+
+        self.fields["jira_project"].choices = projects_list
+        self.fields["jira_project"].widget.choices = self.fields["jira_project"].choices
+
+        self.fields["issuetype"].choices = HARDCODED_ISSUE_TYPES
+        self.fields["issuetype"].widget.choices = self.fields["issuetype"].choices
+
     def clean(self):
         return super(JiraNotifyServiceForm, self).clean()
 
 
+# TODO instead of hard-coding things, lets automatically select the first integration.
+HARDCODED_ISSUE_TYPES = [
+    ("Bug", "Bug"),
+    ("Issue", "Issue"),
+    ("Task", "Task"),
+]
+
+
 class JiraCreateTicketAction(IntegrationEventAction):
     form_cls = JiraNotifyServiceForm
-    label = u"TODO Create a {name} Jira ticket"
+    label = u"TODO Create a {jira_integration} {jira_project} Jira ticket, name: {name} type: {issuetype}!"
     prompt = "Create a Jira ticket"
     provider = "jira"
-    integration_key = "jira_project"
+    integration_key = "jira_integration"
 
     def __init__(self, *args, **kwargs):
         super(JiraCreateTicketAction, self).__init__(*args, **kwargs)
-        # TODO 1.1 Add form_fields
-        self.form_fields = {}
+        all_integrations = self.get_integrations()
+        integration_choices = [(i.id, i.name) for i in all_integrations]
+
+        self.form_fields = {
+            "jira_integration": {
+                "type": "choice",
+                "choices": integration_choices,
+                "default": integration_choices[0][0],
+                "updatesForm": True,
+            },
+            "jira_project": {
+                "type": "choice",
+                "choices": integration_choices,
```

Comment:
@ceorourke why are these getting the choices of your Jira integration installations instead of the projects in your Jira?